client/web/src/components/externalServices/ExternalServiceEditPage.tsx

```diff
@@ -111,50 +157,7 @@ export const ExternalServiceEditPage: FC<Props> = ({
                 <PageTitle title="Code host" />
             )}
             {combinedError !== undefined && !combinedLoading && <ErrorAlert className="mb-3" error={combinedError} />}
-            {externalService && (
-                <Container className="mb-3">
-                    <PageHeader
-                        path={path}
-                        byline={
-                            <CreatedByAndUpdatedByInfoByline
-                                createdAt={externalService.createdAt}
-                                updatedAt={externalService.updatedAt}
-                                noAuthor={true}
-                            />
-                        }
-                        className="mb-3"
-                        headingElement="h2"
-                        actions={
-                            <ButtonLink
-                                to={`/site-admin/external-services/${encodeURIComponent(externalService.id)}`}
-                                variant="secondary"
-                            >
-                                Cancel
-                            </ButtonLink>
-                        }
-                    />
-                    {externalServiceCategory && (
-                        <ExternalServiceForm
-                            input={{ ...externalService }}
-                            externalServiceID={externalServiceID}
-                            editorActions={externalServiceCategory.editorActions}
-                            jsonSchema={externalServiceCategory.jsonSchema}
-                            error={updateExternalServiceError}
-                            warning={externalService.warning}
-                            mode="edit"
-                            loading={combinedLoading}
-                            onSubmit={onSubmit}
-                            onChange={onChange}
-                            telemetryService={telemetryService}
-                            autoFocus={autoFocusForm}
-                            externalServicesFromFile={externalServicesFromFile}
-                            allowEditExternalServicesWithFile={allowEditExternalServicesWithFile}
-                            additionalFormComponent={externalServiceCategory.additionalFormComponent}
-                        />
-                    )}
-                    <ExternalServiceWebhook externalService={externalService} className="mt-3" />
-                </Container>
-            )}
+            {externalService && renderService(externalService)}
```

Comment:
Instead of having a method called `renderService`, I think a better convention would be to create a component (e.g. rename the function `renderService` to `ExternalServiceContainer` and change the type of it to `React.FC<Props>`. This code would then change to:
```suggestion
            {externalService && <ExternalServiceContainer externalService={externalService} />}
```